src/test/java/com/thomashan/coup/action/ActionDetectorTest.java

```diff
@@ -14,83 +14,83 @@
 import static com.thomashan.coup.action.MainActionType.EXCHANGE;
 import static com.thomashan.coup.action.MainActionType.STEAL;
 import static com.thomashan.coup.action.MainActionType.TAX;
-import static java.util.Collections.singletonList;
+import static java.util.Collections.singleton;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-class MainActionDetectorTest {
+class ActionDetectorTest {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This class has too many methods, consider refactoring it.](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)